puppet/modules/redmine/files/git_repos.sh

```diff
@@ -36,6 +36,10 @@ fi
 # Sync repositories for all known git repos
 curl -s $REDMINE_REPOS | ruby -rjson -e '
 JSON.load(STDIN).each do |project_name,repos|
+  unless repos
```

Comment:
I think you're chasing the wrong `each`. The error I see in my INBOX is 
```
-e:2:in `<main>': undefined method `each' for nil:NilClass (NoMethodError)
```
*line 2*, but `repos.each` is on line 3 (the first line being the empty line at `ruby -rjson -e '`).

I think the failing `each` is `JSON.load(STDIN).each`. Test:

```
% cat /tmp/test.sh 
#!/bin/sh

echo "null" | ruby -rjson -e '
JSON.load(STDIN).each do |p|
	puts p
end'

% bash /tmp/test.sh
Traceback (most recent call last):
-e:2:in `<main>': undefined method `each' for nil:NilClass (NoMethodError)
```